backend/routes.js

```diff
@@ -69,6 +92,17 @@ route.POST('/event', {
         const parsedManifest = JSON.parse(manifest)
         const { name } = JSON.parse(parsedManifest.body)
         if (name !== 'gi.contracts/identity') return Boom.unauthorized('This contract type requires ownership information', 'shelter')
+        // rate limit signups in production
```

Comment:
Commenting here because it's actually a line above not modified by this PR.

```js
const ip = request.headers['x-real-ip'] || request.info.remoteAddress
```

That line needs to have a comment saying that it's a REQUIREMENT that x-real-ip be a trusted header and that if is setting up a server it MUST be set to a trusted value that CANNOT be overridden by users. If that assumption doesn't hold, this PR is useless and potentially dangerous.